x/oracle/keeper/query_server.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	"context"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/provenance-io/provenance/x/oracle/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// QueryAddress returns the address of the module's oracle
+func (k Keeper) OracleAddress(goCtx context.Context, req *types.QueryOracleAddressRequest) (*types.QueryOracleAddressResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
```

Comment:
Since the request doesn't have anything in it, we don't really care whether or not one comes in.
```suggestion
func (k Keeper) OracleAddress(goCtx context.Context, _ *types.QueryOracleAddressRequest) (*types.QueryOracleAddressResponse, error) {
```